.circleci/config.yml

```
@@ -146,3 +145,50 @@ jobs:
                     --attach build/distributions/git-machete-intellij-plugin-*.zip \
                     --message "$tag"$'\n\n'"$change_notes" \
                     $tag
+  checkForNewEapBuildNumber:
+    docker:
+      - image: 'gitmachete/intellij-plugin-ci:6.1.0'
+    working_directory: ~/git-machete-intellij-plugin
+    resource_class: large
+    steps:
+      - checkout
+      - run:
+          name: Reconfigure origin remote to https
+          command: git remote set-url origin https://${GITHUB_TOKEN}@github.com/VirtusLab/git-machete-intellij-plugin.git
+      - run:
+          name: Configure email and name
+          command: |
+            git config user.email "cron_job@example.com"
+            git config user.name "cron_job"
+      - run:
+          # language=sh
+          command: |
+            ./gradlew updateEapBuildNumber
+            if [[ `git status --porcelain` ]]; then
```

Nit: just sth like?

 ```suggestion
            if git diff --exit-code intellijVersion.properties; then
```

So that you can rely just on exit code and not on some output being empty or not (which is generally more brittle)